hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallLocalSuite.java

```diff
@@ -220,7 +219,7 @@ final HapiSpec invalidDeletedContract() {
                 .when(contractDelete(CONTRACT))
                 .then(contractCallLocal(CONTRACT, "create")
                         .nodePayment(1_234_567)
-                        .hasAnswerOnlyPrecheck(CONTRACT_DELETED));
+                        .hasAnswerOnlyPrecheck(OK));
```

Comment:
There's overlap! It doesn't _need_ to be in either suite, it could live in _either_!  The reason to put them in one suite would be to find them in one place.  But then, which suite?  Can't think of a reason to choose one over the other.  And if you can't either ... then leave them alone.

Reply:
Meanwhile, name of test should still be changed.